### PR TITLE
Add step by step instructions on contributing using git

### PR DESCRIPTION
**Pull request by jd-au**

### CODE REVIEW

File: ivoatexDoc.tex [hunk 1]
```diff
@@ -1136,6 +1136,50 @@ \subsection{The ivoa-std Organisation}
...
+\begin{lstlisting}[language=sh]
+git checkout main
+git pull origin main
+git checkout -b branch-name-for-change
+\end{lstlisting}
+
+Once you are happy with the change (and we do recommend building the PDF document locally to review it) you can save it to git and send it to your forked repository.
+
+\begin{lstlisting}[language=sh]
+git commit
```
Comment (molinaro-m) [inline ivoatexDoc.tex:1176]: Should we suggest a
`git commit -m "a useful short description of my changes`
as well as a reminder on the `git add whta files you changed` ?

Or would that be too much basic detail?

**Review by molinaro-m — APPROVED** [review]
I like the text as well as the triangle diagram.
I added a small comment that is not critical to the approval , IMHO.

**Review by msdemlei — CHANGES_REQUESTED** [review]
In general, I'd ask everyone to author with ~72 characters per line (and not reflowing for general commits) -- it makes later diffs much easier to comprehend.  You know, it's source code...

Other than that, would you add triangle_workflow.png to FIGURES in the makefile?

File: ivoatexDoc.tex (hunk 1 above)
Comment (jd-au) [reply ivoatexDoc.tex]: I think we should, particularly as I forgot to include the Makefile in my first commit! I've made those changes.

**Comment by jd-au** [comment]
> In general, I'd ask everyone to author with ~72 characters per line (and not reflowing for general commits) -- it makes later diffs much easier to comprehend. You know, it's source code...
> 
> Other than that, would you add triangle_workflow.png to FIGURES in the makefile?

**Review by msdemlei — APPROVED** [review]

**Pull request merged**